### PR TITLE
Fix warning

### PR DESCRIPTION
**Pull request by hwchiu**

### CODE REVIEW

File: tools/python/src/face_recognition.cpp [hunk 1]
```diff
@@ -166,7 +166,6 @@ boost::python::list chinese_whispers_clustering(boost::python::list descriptors,
                 edges.push_back(sample_pair(i,j));
         }
     }
-    const auto num_clusters = chinese_whispers(edges, labels);
```
Comment (davisking) [inline tools/python/src/face_recognition.cpp]: ?????

**Comment by davisking** [comment]
You need to run the code to make sure it works before submitting a PR.  I'm saying this because this change obviously breaks the code.

File: tools/python/src/face_recognition.cpp (hunk 1 above)
Comment (hwchiu) [reply tools/python/src/face_recognition.cpp]: Sorry for my mistake. I will fix it.
And I want to know if we don't care the return value of `chinese_whispers`, do we need this variable ?

Comment (davisking) [reply tools/python/src/face_recognition.cpp]: Variables that aren't used aren't needed.  But the function call is obviously hugely important.

Comment (hwchiu) [reply tools/python/src/face_recognition.cpp]: Thanks, I have add it back.

**Comment by davisking** [comment]
Looks good, thanks for the PR :)

**Pull request merged**